### DISCUSSION TITLE
"Windows protected your PC"

### DISCUSSION THREAD

**Discussion by avi12** [kimlimjustin/xplorer#212] [Q&A] [answered]
The prompt makes me feel uneasy
If Xplorer could make it go away (e.g. by having the required certificate), that'd be great
![image](https://user-images.githubusercontent.com/6422804/148982754-001695a8-220d-4bbd-a3d5-7864573433b2.png)

**Comment by lg-nineoneninethreenine** [comment]
Yeah but you need to pay for a cert. I think that's hard.

**Comment by kimlimjustin** [comment, marked as answer]
As @im-coder-lg mentioned, you just can't disable that as Windows also try to protect non-techy users from unwanted software, what you can do is to [run it anyway](https://xplorer.space/docs/install/#common-problems).

The way we can avoid that is to whether buy a certificate, or to submit it to Microsoft store, but both takes hundreds of bucks, to get so, please consider [sponsor us](https://opencollective.com/xplorer).

**Reply by avi12** [reply]
It seems like [Microsoft](https://developer.microsoft.com/en-us/microsoft-store/register/) only wants a single payment of $99 (assuming it's a company account)
![image](https://user-images.githubusercontent.com/6422804/149081042-34a55f34-c667-47e6-a910-e7bac393cfa8.png)

At least it's not for Mac App Store which demands a fee of $99 **per year**

**Reply by kimlimjustin** [reply]
I know it might seem little to you but it's somehow much for students like me to afford it. Hence, if you really wanna support us, please consider[ sponsoring us](https://opencollective.com/xplorer) and I'll use it for the submission cost.

**Reply by avi12** [reply]
$99 does not seem little to me, but $19 does seem far less
